Network/Socket.hsc

```diff
@@ -1021,7 +1027,10 @@ aNY_PORT = 0
 iNADDR_ANY :: HostAddress
 iNADDR_ANY = htonl (#const INADDR_ANY)
 
+-- | Converts the from host byte order to network byte order.
```

Comment:
It might be helpful to enumerate the possible `id` behaviour of these functions.